apps/io-services-cms-webapp/src/watchers/on-legacy-service-change.ts

```diff
@@ -192,3 +192,13 @@ export const handler =
       ),
       O.getOrElse(() => TE.right(noAction))
     );
+
+const calculateServiceName = (serviceName: NonEmptyString) => {
+  const calculatedName = serviceName.replace("DELETED", "").trim();
+
+  if (NonEmptyString.is(calculatedName)) {
+    return calculatedName;
+  }
+
+  return "-" as NonEmptyString;
```

Comment:
another way is 

`return NonEmptyString.decode(calculatedName).getOrElse(() => "-")`